src/driver-holtek.h

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+/*
+ * Copyright © 2023 Michał Lubas
+ */
+
+#pragma once
+
+#include "libratbag-util.h"
+
+#define HOLTEK8_FW_VERSION_LEN 4
+
+enum holtek8_sensor {
+	HOTLEK8_SENSOR_UNKNOWN = 0,
+	HOTLEK8_SENSOR_PAW3333,
+	HOTLEK8_SENSOR_PMW3320,
+};
+
+struct holtek8_device_data {
+	char *fw_version;
+	char *device_name;
+	enum holtek8_sensor sensor;
+	int button_count;
+	uint8_t password[6];
```

Comment:
Make it [7] and you can get nice null termination ;)

Reply:
This one was kind of on purpose because at this point I started treating it as an array of 6 bytes instead of a string, but on a second thought these passwords are always ASCII so we may store a null terminator to make for example a debug printf much easier with %s. Let me know which way you prefer.

Reply:
It's uint8_t[] so it's clear it's not a string, so it's good. :)